### ISSUE TITLE
Inception resnet V2 weird training behavior

### ISSUE THREAD

**Issue by Simanas** [tflearn/tflearn#874]
I tried to train inception resnet V2 from here: https://github.com/tflearn/tflearn/blob/master/examples/images/inception_resnet_v2.py

I did my little modifications on input data dimensions as well as on output layer. Then I try to train it on the same data I have successfully trained inception V3. I have relatively unlimited size training set. Now when train loss goes down pretty nicely, a little bit faster then with inception V3 (that is expected), but my validation loss goes nowhere. It just stays at the same starting position and basically just fluctuates a little bit in place. It's not going up and not going down. Is there anybody who could give me some hint on what I do not understand here? This is super mysterious...
